plugins/plugins.go

```diff
@@ -194,6 +225,42 @@ func New(raw []byte, id string, store storage.Store, opts ...func(*Manager)) (*M
 	return m, nil
 }
 
+// Init returns an error if the manager could not initialize itself. Init() should
+// be called before Start().
+func (m *Manager) Init(ctx context.Context) error {
```

Comment:
Does it make sense to make it idempotent or raise an error on subsequent calls? Im guessing we don't want to let it wipe the compiler and add another commit trigger if someone calls init >1 time... or maybe we do?

Reply:
Fixed. Made it be idempotent by checking the initialized flag.